src/AppInstallerCLICore/ExecutionContext.h

```diff
@@ -122,6 +122,9 @@ namespace AppInstaller::CLI::Execution
 
         virtual void SetExecutionStage(Workflow::ExecutionStage stage, bool);
 
+        // Get Globals for Current Thread
+        AppInstaller::ThreadLocalStorage::ThreadGlobals& GetThreadGlobalsActive();

```

Comment:
This has nothing to do with threads; it is simply the `Context`'s `ThreadGlobals` object:
```suggestion
        AppInstaller::ThreadLocalStorage::ThreadGlobals& GetThreadGlobals();
```